.github/workflows/ci.yml

```diff
@@ -175,6 +178,41 @@ jobs:
       - name: Test
         run: ./ci/scripts/run-english-breakfast-verilator-tests.sh
 
+  # Build CW305 variant of the English Breakfast toplevel design using Vivado
+  chip_englishbreakfast_cw305:
+    name: CW305's Bitstream
+    runs-on: ubuntu-22.04-bitstream
+    needs: quick_lint
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        uses: ./.github/actions/install-deps
+      - name: Build bitstream
+        run: |
+          . util/build_consts.sh
+
+          # Build CW305 test rom required by `build-bitstream-vivado.sh`
+          rom_path="sw/device/lib/testing/test_rom"
+          ci/bazelisk.sh build "//${rom_path}:test_rom_fpga_cw305" \
+            --features=-rv32_bitmanip \
+            --copt=-DOT_IS_ENGLISH_BREAKFAST_REDUCED_SUPPORT_FOR_INTERNAL_USE_ONLY_
+          vmem="$(ci/bazelisk.sh cquery --output=files "//${rom_path}:test_rom_fpga_cw305" \
+            --features=-rv32_bitmanip \
+            --copt=-DOT_IS_ENGLISH_BREAKFAST_REDUCED_SUPPORT_FOR_INTERNAL_USE_ONLY_
+          )"
+          mkdir -p "build-bin/${rom_path}"
+          cp "$vmem" "build-bin/${rom_path}"
+          realpath "build-bin/${rom_path}"
```

Comment:
I think this was a debug logging line that I left in by mistake, you can probably remove it